### PR TITLE
APIGOV15234 Security and Network topics

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [x] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Comment by lbadenhop** [comment]
@jcollins-axway and @dfeldick,
Please review before @cbordier merges with master.

File: content/en/docs/central/connect-api-manager/network-traffic-apimanager.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+* Internal:
+
+    * `8075/tcp`: default port for communication to API Manager
+
+    * `8090/tcp`: default port for communication to API Gateway
+
+* Inbound (used for the agent status server):
+
+    * `8989/tcp`: default port for serving the agent status
```
Comment (jcollins-axway) [inline content/en/docs/central/connect-api-manager/network-traffic-apimanager.md:42]: All of these port sections were tables in the New doc on the ticket. It includes Host, Port, Protocol, and Data columns

File: content/en/docs/central/connect-api-manager/network-traffic-apimanager.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+A proxy can be set within the configuration file for all connections made by the agents.
+
+### HTTP/HTTPS Proxy
+
+Use a HTTP/HTTPS Proxy for communication to the AMPLIFY Platform.  This configuration is set for both the [Discovery](/docs/central/discovery-agent-variables/) and [Traceability Agents](/docs/central/traceability-agent-variables/).
+
+### SOCKS5 Proxy
+
+Use a SOCKS5 Proxy for communication to the AMPLIFY Platform when sending API Traffic Events.  This configuration is set only for [Traceability Agents](/docs/central/traceability-agent-variables/).
```
Comment (jcollins-axway) [inline content/en/docs/central/connect-api-manager/network-traffic-apimanager.md:54]: There was a Proxy Authentication section added in the latest doc as well

File: content/en/docs/central/connect-api-manager/network-traffic-apimanager.md (hunk 1 above)
Comment (lbadenhop) [reply content/en/docs/central/connect-api-manager/network-traffic-apimanager.md:42]: @jcollins-axway  -- I must have pulled down the wrong file. I'll update it.

File: content/en/docs/central/connect-aws-gateway/network-traffic-aws.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+All outbound traffic is sent over SSL via TCP / UDP.
+
+Open the following ports to benefit from all the Agent functionalities:
+
+* Outbound:
+
+    * `443/tcp`: port for communication to AMPLIFY Central and AWS
+
+    * `453/tcp`: port for all API event data
```
Comment (jcollins-axway) [inline content/en/docs/central/connect-aws-gateway/network-traffic-aws.md:30]: The table representation of this and the Inbound section seem to be missing here as well

**Comment by lbadenhop** [comment]
@jcollins-axway --please look over your topics. I've added the tables and missing content. 
@cbordier --ready for your review. You can merge to master once Jason and Dale review.

File: content/en/docs/central/connect-api-manager/network-traffic-apimanager.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+### Proxy authentication
+
+Both proxy types will use one or two authentication mechanisms, none and username/password authentication. The username authentication is specified within the URL `http://{user}:{pass}@{proxy}:{port}`.
```
Comment (jcollins-axway) [inline content/en/docs/central/connect-api-manager/network-traffic-apimanager.md:63]: my mistake here, should say ....one **of** two authentication, none **or** username/password..."

Comment (lbadenhop) [reply content/en/docs/central/connect-api-manager/network-traffic-apimanager.md]: fixed

**Comment by lbadenhop** [comment]
Hi @cbordier-- these topics are ready for you to review and push to master. Once done:

- Add a comment for Alex and Andrea to upload to docs.axway.com 
- Reset the label to "needs push to prod"  
- Close the Docs-Review Jiras with comment that the topic has been reviewed, merged with master and is waiting for upload to docs.away.com
 
Thanks

**Comment by cbordier** [comment]
@alexearnshaw , @andreamussap : this PR is ready to be upload to docs.axway.com

**Pull request merged**